amass/config.go

```diff
@@ -186,3 +218,108 @@ func (c *Config) GetAPIKey(source string) *APIKey {
 	}
 	return nil
 }
+
+// LoadSettings parses settings from an .ini file and assigns them to the Config.
+func (c *Config) LoadSettings(path string) error {
+	cfg, err := ini.LoadSources(ini.LoadOptions{
+		Insensitive:  true,
+		AllowShadows: true,
+	}, path)
+	if err != nil {
+		return err
+	}
+	// Get the easy ones out of the way using mapping
+	if err = cfg.MapTo(c); err != nil {
+		return err
+	}
+	// Attempt to load a wordlist provided via the configuration file
+	if cfg.Section(ini.DEFAULT_SECTION).HasKey("wordlist_file") {
+		wordlist := cfg.Section(ini.DEFAULT_SECTION).Key("wordlist_file").String()
+
+		if list, err := getWordList(wordlist); err == nil {
+			c.Wordlist = list
+		} else {
+			return err
+		}
```

Comment:
The if..else is usually frowned upon in Go, the following the preferable:

```
list, err := getWordList(wordlist)
if err != nil {
    return err
}

c.Wordlist = list
```

this is true below as well.

Reply:
Fixed those up